pyproject.toml

```diff
@@ -24,7 +24,8 @@ docs = ["autoapi",
     "sphinx-collections",
     "sphinx-copybutton",
     "sphinx-gallery",
-    "pydata-sphinx-theme"
+    "pydata-sphinx-theme",
+    "pandas"
```

Comment:
note: we had to add pandas here since we are converting the astropy table to pandas in order to get the json safe dictionary of user defined keys to search on. This is the best way I've found to do this, but I'm open to any suggestions